client/wordpress-com.js

```diff
@@ -369,4 +369,12 @@ sections.push( {
 	secondary: true
 } );
 
+sections.push( {
+	name: 'domain-connect-authorize',
+	paths: [ '/domain-connect/authorize' ],
+	module: 'my-sites/upgrades/domain-management/domain-connect',
+	enableLoggedOut: false,
```

Comment:
Yep. Really good catch. This was the result of a conflict introduced when I was cherry-picking. Oops!